Geometry/VeryForwardGeometry/python/geometryRP_cfi.py

```diff
@@ -81,8 +81,38 @@
         'Geometry/VeryForwardData/data/RP_Cuts_Per_Region.xml',
         'Geometry/VeryForwardData/data/RP_Param_Beam_Region.xml')
 
+ctppsDiamondGeomXMLFiles = cms.vstring(
```

Comment:
Note that 91X does not have the following in totemGeomXMLFiles
```xml
       'Geometry/CMSCommonData/data/PhaseI/beampipe.xml',
       'Geometry/CMSCommonData/data/cmsBeam.xml',
       'Geometry/CMSCommonData/data/mgnt.xml',
```
IIRC, some were not supposed to be included. 
Should that cleanup be added here as well?